app/Filament/Resources/ServerResource/Pages/ListServers.php

```diff
@@ -81,12 +80,7 @@ public function table(Table $table): Table
                 Action::make('View')
                     ->icon('tabler-terminal')
                     ->url(fn (Server $server) => "/server/$server->uuid_short")
-                    ->visible(function (Server $server) {
-                        /** @var User $user */
-                        $user = auth()->user();
-
-                        return $user->isRootAdmin() || $user->id === $server->owner_id;
-                    }),
+                    ->visible(fn () => auth()->user()->can('view server')),
```

Comment:
```suggestion
                    ->authorize(fn () => auth()->user()->can('view server')),
```

Reply:
Just noticed the _View_ button leads to the react part which doesn't know what to do with Roles should i mke it aware of it or should we wait for the Filament frontend ?
Also rows in ListServers are not clickable if not rootAdmin